### DISCUSSION TITLE
Doble consulta sobre las practicas resueltas en cursada2023, Practica 1 y 2.

### DISCUSSION THREAD

**Discussion by LautaroTummino** [INGCOM-UNRN-P1/cursada-2023#146] [Preguntas y respuestas] [answered]
¿Qué está sucediendo?. No entendo nada.
![](https://cdn.discordapp.com/attachments/1141205473190760498/1149818103153303622/image.png)

**Comment by LautaroTummino** [comment]
Estamos revisando las prácticas resueltas con @AnGys09 y observamos que tienen algunos errores en cuanto a el uso de las llaves { como también,  en el TP1 Ejercicio 4 que se está definiendo MAYOR, MENOR e IGUAL, pero no se le da uso en la función. 

Tengo entendido se puede hacer la corrección dentro del código, o dejar un comentario verdad?

Saludos profe!

**Reply by LautaroTummino** [reply]
Ahhh ahí me explico Angy como hacer para dejar comentarios dentro de TP. 
Lo vimos en clase pero me olvide.

**Comment by AnGys09** [comment]
ahora la gran pregunta es como marcamos como resuelto el post :p

**Reply by LautaroTummino** [reply]
Arriba a la derecha te dice review changes.
Ahí 'creo' dejas el comentario y le das a submit review para completar.

![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/105172727/49c71062-7846-4c5f-940c-476281a88a48)

**Reply by martinvilu** [reply]
Sactamente

**Comment by martinvilu** [comment, marked as answer]
> ¿Qué está sucediendo?. No entendo nada.

Es un arreglo doble, primero tenés, dos valores para argumentos y un tercero para el resultado.
Y después, un conjunto de estos conjuntos de valores para hacerlo tantas veces como "casos" quieras probar.

Se llama a la función con esos valores y se comparan con el resultado esperado para validar que la función esté haciendo las cuentas correctamente.
Uno por caso.

Específicamente _que_ es algo que vamos a ver entre el miércoles próximo y el otro.

**Comment by LautaroTummino** [comment]
Edite la 2da consulta ya que logre hacerlo desde el pull request, y fue resuelto! 
Gracias profe!
